TrialOfValor/Helya.lua

```diff
@@ -155,13 +181,16 @@ end
 
 function mod:OnEngage()
 	taintMarkerCount = 4
-	orbCount = 1
-	tentaclesUp = 9
+	tentaclesUp = self:Mythic() and 8 or 9
+	orbCounter = 1
 	phase = 1
-	self:Bar(227967, 12) -- Bilewater Breath
-	self:Bar(228054, 19.5) -- Taint of the Sea
-	self:Bar(229119, 31, CL.count:format(self:SpellName(229119), orbCount)) -- Orb of Corruption
-	self:Bar(228730, 37) -- Tentacle Strike
+	tentacleCounter = 1
+	
+	self:Bar(227967, self:Mythic() and 10.5 or 12) -- Bilewater Breath
+	self:Bar(228054, self:Mythic() and 15.5 or 19.5, CL.count:format(self:SpellName(228054), taintCount)) -- Taint of the Sea
+	self:Bar("orb_ranged", self:Mythic() and 14 or 31, CL.count:format(L.orb_bar:format("Ranged"), orbCounter), 229119) -- Orb of Corruption
+	self:Bar(228730, self:Mythic() and 35.3 or 36.7, CL.count:format(self:SpellName(228730), tentacleCounter)) -- Tentacle Strike
+	self:Mythic() and self:Berserk(660)
```

Comment:
This isn't valid syntax, which is why the build is failing. Should be an if statement.